modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -208,53 +210,6 @@ const StyledBox = styled(Box)({
 <Box fontFamily="monospace" fontSize={16} fontWeight="medium" />
 ```
 
-## General Changes
```

Comment:
This section was out of order from the Table of Contents at the top, so I moved it.